index.js

```diff
@@ -9,9 +11,35 @@
  *
  */
 
+var lib = require('manifoldjs-lib');
+var libValidationContants = lib.constants.validation;
+
+// Maintain compatibility with original constant definitions
+var validationConstants = {
+  levels:           libValidationContants.levels,
```

Comment:
Nit: maybe get rid of the white-space for the following three?... the ones below don't use white-space... it's also a maintenance burden.